docs/src/reference/asciidoc/index.adoc

```diff
@@ -27,18 +27,26 @@
 
 {ehtm} is an `umbrella' project consisting of three similar, yet independent sub-projects with their own, dedicated, section in the documentation:
 
-{eh} proper:: interact with {es} from within a Hadoop environment. If you are using {mr}, Hive, Pig, {sp}, {st}, or Cascading, this project is for you. For feature requests or bugs, please open an issue in the https://github.com/elastic/elasticsearch-hadoop/issues[Elasticsearch-Hadoop repository].
+{eh} proper::
+Interact with {es} from within a Hadoop environment. If you are using {mr}, Hive, Pig, {sp}, {st}, or Cascading, this
+project is for you. For feature requests or bugs, please open an issue in the
+https://github.com/elastic/elasticsearch-hadoop/issues[Elasticsearch-Hadoop repository].
 
-repository-hdfs:: use HDFS as a back-end repository for doing snapshot/restore from/to {es}. For more information, refer to its https://www.elastic.co/guide/en/elasticsearch/plugins/master/repository-hdfs.html[home page]. For feature requests or bugs, please open an issue in the https://github.com/elastic/elasticsearch/issues[Elasticsearch repository] with the ":Plugin Repository HDFS" tag.
+repository-hdfs::
+Use HDFS as a back-end repository for doing snapshot/restore from/to {es}. For more information, refer to its
+https://www.elastic.co/guide/en/elasticsearch/plugins/master/repository-hdfs.html[home page]. For feature requests or
+bugs, please open an issue in the https://github.com/elastic/elasticsearch/issues[Elasticsearch repository] with the
+":Plugin Repository HDFS" tag.
 
-{ey}:: run {es} on top of YARN - see <<es-yarn>>. This project is in beta.
+{ey} [DISCONTINUED]::
```

Comment:
@acchen97 I don't have a strong opinion either way since I don't own this doc. But I think your point could be made for a lot of features that we remove from the docs over time. Docs can get pretty bloated if we keep stuff like this around, and people forget to remove it. If we are  good about flagging deprecation early on and reminding users that the feature is going away, statements like this are unnecessary, IMO. While useful to users in the short term, these statements represent obsolete content that we need to maintain over time. IMO, it's better to train users to look in the release notes. I've been a tech writer for 20+ years so I've seen a lot of stuff like this die on the vine over the years. Anyhow, just something to keep in mind for the future.